src/prefect/_internal/pydantic/_flags.py

```diff
@@ -1,11 +1,15 @@
 import os
 
+# Retrieve current version of Pydantic installed in environment
 from pydantic.version import VERSION as PYDANTIC_VERSION
 
+# Check if Pydantic version 2 is being used
```

Comment:
just because the other flag is called `USE_PYDANTIC_V2` 😅 
```suggestion
# Check if Pydantic version 2 is the installed version
```

Reply:
yaaas